src/device-registry/config/constants.js

```diff
@@ -1234,6 +1250,8 @@ const defaultConfig = {
       "devices.site.distance_to_nearest_residential_road": 0,
       "devices.site.bearing_to_kampala_center": 0,
       "devices.site.distance_to_kampala_center": 0,
+      "devices.site.bearing_to_capital_city": 0,
```

Comment:
@BenjaminSsempala , could we consider *`_to_capital_city_center`?

Reply:
Yes @Baalmart I've updated this.